compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -52,11 +53,13 @@ import qualified DA.Daml.LF.TypeChecker.Env as LF
 import qualified DA.Daml.LF.TypeChecker.Error as LF
 import qualified DA.Daml.LFConversion.MetadataEncoding as LFC
 import DA.Daml.Options
+import DA.Daml.UtilGHC (fsFromText)
 
 import SdkVersion
 
-panicOnError :: Either LF.Error a -> a
-panicOnError (Left e) = error $ "Internal LF type error: " <> renderPretty e
+panicOnError :: HasCallStack => Either LF.Error a -> a
```

Comment:
also not crucial, and not even related, but gives us slightly better error messages